src/services/github.service.spec.ts

```diff
@@ -102,16 +109,19 @@ describe('Test GithubService', () => {
       .spyOn(httpService, 'get')
       .mockImplementationOnce(() => throwError(err));
 
-    try {
-      await githubApiService.getUser('usernameTest', mockHeaders);
-      fail('should throw');
-    } catch (e) {
-      expect(e.status).toEqual(404);
-      expect(e.message).toEqual('GitHub user not found');
-    }
+    githubApiService.getUser('vitalii', mockHeaders).subscribe(
+      (next: User) => {
+        done.fail('should throw 404 Error');
+      },
+      (error: any) => {
```

Comment:
How come it's `any`, yet You expect `status` and `message` fields to be there? :) Add proper types